collectors/VCenterPropertiesCollector.py

```diff
@@ -0,0 +1,53 @@
+from BaseCollector import BaseCollector
+import os, time, json
+from prometheus_client.core import GaugeMetricFamily
+from tools.Resources import Resources
+from tools.YamlRead import YamlRead
+
+class VcenterPropertiesCollector(BaseCollector):
+    def __init__(self):
+        self.wait_for_inventory_data()
+        self.property_yaml = YamlRead('collectors/property.yaml').run()
+        self.g = GaugeMetricFamily('vrops_vcenter_properties', 'testtest',

```

Comment:
please configure this in collect(), have a glimpse at the other modules, this has changed

Reply:
not sure about this, let me check and get back to you